test/functional/Java16andUp/playlist.xml

```diff
@@ -116,6 +116,32 @@
 			-excludegroups $(DEFAULT_EXCLUDE); \
 			$(TEST_STATUS)
 		</command>
+		<platformRequirements>bits.64,^arch.ppc,^arch.390,^arch.riscv,^os.zos,^os.sunos,^arch.arm</platformRequirements>
```

Comment:
just to double check, is it intended to add `platformRequirements` for `Jep397Tests_testSubClassOfSealedSuperFromDifferentPackageInSameNamedModule`?

Reply:
The `platformRequirements` is temporarily set up for JEP389 (which is irrelevant to JEP397) to disable a couple of platforms we have not yet implemented but will do in the future as scheduled.